tests/bench/string-interpolation-macro/Macro.scala

```diff
@@ -0,0 +1,18 @@
+import scala.quoted._
+
+object Macro {
+
+  extension (inline sc: StringContext):
+    inline def x(inline args: Int*): String = ${ code('sc, 'args) }
+
+  def code(strCtxExpr: Expr[StringContext], argsExpr: Expr[Seq[Int]])(using QuoteContext): Expr[String] =
+    var res: Expr[String] = null
+    for _ <- 0 to 1_000_000 do
```

Comment:
I still need to fine-tune this parameter. @liufengyun how can I run this benchmark locally using `dotty-bench/jmh:run`? I need to test the compilation of the `Test.scala` file with this one on the classpath.

Reply:
You can run it as follows:
```
sbt > dotc -d out tests/bench/string-interpolation-macro/Macro.scala

sbt > dotty-bench-bootstrapped/jmh:run -classpath out tests/bench/string-interpolation-macro/Test.scala
```

Reply:
It seems that it is ignoring the classpath
```
[info] running (fork) dotty.tools.benchmarks.Bench -classpath out tests/bench/string-interpolation-macro/Test.scala
# JMH version: 1.19
# VM version: JDK 1.8.0_265, VM 25.265-b01
# VM invoker: /Library/Java/JavaVirtualMachines/adoptopenjdk-8.jdk/Contents/Home/jre/bin/java
# VM options: -Xms2G -Xmx2G
# Warmup: 30 iterations, 1 s each
# Measurement: 20 iterations, 1 s each
# Timeout: 10 min per iteration
# Threads: 1 thread, will synchronize iterations
# Benchmark mode: Average time, time/op
# Benchmark: dotty.tools.benchmarks.Worker.compile

# Run progress: 0.00% complete, ETA 00:00:50
# Fork: 1 of 1
# Warmup Iteration   1: -- [E006] Not Found Error: ../tests/bench/string-interpolation-macro/Test.scala:1:7 
1 |import Macro._
  |       ^^^^^
  |       Not found: Macro

longer explanation available when compiling with `-explain`
-- [E008] Not Found Error: ../tests/bench/string-interpolation-macro/Test.scala:4:21 
4 |  def test: String = x"a${1}b${2}c${3}d${4}e${5}f"
  |                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |        value x is not a member of StringContext.
  |        An extension method was tried, but could not be fully constructed:
  |
  |            <error Not found: Macro>#extension_x()
```

Reply:
It needed `-classpath ../out`